crates/sui-rpc-loadgen/src/main.rs

```diff
@@ -79,6 +79,17 @@ pub enum ClapCommand {
         #[clap(flatten)]
         common: CommonOptions,
     },
+    #[clap(name = "query-transactions")]
+    QueryTransactions {
+        #[clap(short, long)]
+        from_address: Option<String>,
+
+        #[clap(short, long)]
+        to_address: Option<String>,
+
+        #[clap(flatten)]
+        common: CommonOptions,
```

Comment:
I think what will be more useful for the purpose of loadgen testing is to add an option for providing a `path` to a file that contains a giant list of addresses(generated by `getCheckpoints`).

Reply:
Makes sense, I was going to implement this for `getOwnedObjects` but can see how it would apply to `queryTransactions` too